toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -1042,6 +1049,16 @@ void CellArea::drawLevelCell(QPainter &p, int row, int col, bool isReference) {
   TXshCell cell = xsh->getCell(row, col);
   TXshCell prevCell;
 
+  // If the user change the option to start a sequence with 0 to 1 in-live, we
+  // delete all cell and frame 0 of the actual Xsheet
+  if (!Preferences::instance()->isSequenceCanStartWith0() &&
+      cell.getFrameId().getNumber() == 0) {
+    cell.getSimpleLevel()->eraseFrame(TFrameId(0));
```

Comment:
I agree that we have to think how to handle the 0-start levels in the environment WITHOUT the 0-start option. 

You are erasing the frame data here only in the drawing function of Xsheet.
But if user works without opening the Xsheet panel, this operation will not be called.
I think "controlling data" operations like this should not be in "viewing data" function.

How about to making this option to demand users to restart the software, 
and moving this operation to the moment when when loading levels?
By restarting the software, you don't need to take care of the existing scene.

In addition, IMO it would be better to open some confirmation dialog before erasing the frame 0 data, 
which enables users to choose whether to erase frame 0 or to activate the preference option.